javascript/URDFLoader.js

```diff
@@ -94,26 +94,33 @@ class URDFLoader {
 
         // Check if a full URI is specified before
         // prepending the package info
+
         let path = urdf;
-        if (!/^[^:]+:\/\//.test(path)) {
+
+        if( !pkg.includes(",") && (!/^[^:]+:\/\//.test(path))) {
 
             // make sure we don't insert a double slash by cleaning
             // the package and urdf paths
             path = `${ pkg.replace(/(\\|\/)$/, '') }/${ urdf.replace(/^(\\|\/)/, '') }`;
-
         }
+        // Convert pkg(s) into a map of type
+        // {"pkg_name": "path/to/pkg_name"}
+        this.pkgMap = pkg.split(",").reduce(function(map, value) {
```

Comment:
I think the package map should be passed down into the parse function the way it was before. If we keep it on the loader object then you'll run into an issue when loading two URDF models back to back:

```js
loader.load({ package1: '.../package1' }, '.../path.urdf');
loader.pkgMap; // this contains package1 only

loader.load({ package2: '.../package2' }, '.../path.urdf');
loader.pkgMap; // this contains package2 only, but the first URDF file is still downloading
```

Reply:
ok I will change that back